src/main/java/Application/OpenCVServer/OpenCVServerOperator.java

```diff
@@ -0,0 +1,164 @@
+
+    public static OpenCVServerOperator getInstance() {
+        if (instance == null) {
+            instance = new OpenCVServerOperator();
+        }
+        return instance;
+    }
+
+    public void setupServerRunners(String args[]) throws IllegalArgumentException {
+        String[] argument;
+        String masterUrl = ""; // for test: http://host.docker.internal:4567/
+        String beatCommand = ""; // for test: client/register
+        String serverId = "";
+        String serverIp = "";
+        boolean connected = false;
+
+        serverRunner = new ServerRunner();
+
+        int missingParameter = 0;
+        String missingParameterList = "";
+
+        for(int i=0; i<args.length; i++) {
+            argument = args[i].split("=");
+            if(argument.length == 2) {
+                switch (argument[0]) {
+                    case "SERVER_ID":
+                        serverId = argument[1];
+                        break;
+                    case "SERVER_IP":
+                        serverIp = argument[1];
+                        break;
+                    case "MASTER_URL":
+                        masterUrl = argument[1];
+                        break;
+                    case "BEAT_COMMAND":
+                        beatCommand = argument[1];
+                        break;
+                    case "CONNECTED":
+                        connected = Boolean.parseBoolean(argument[1]);
+                        break;
+                    case "PORT":
+                        port = Integer.parseInt(argument[1]);
+                        break;
+                    default:
+                        throw new IllegalArgumentException("Invaild argument");
+                }
+            }
+        }
+        if(serverId.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "SERVER_ID,";
+        }
+        if(serverIp.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "SERVER_IP,";
+        }
+        if(masterUrl.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "MASTER_URL,";
+        }
+        if(beatCommand.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "BEAT_COMMAND,";
+        }
+        if(port == 0) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "PORT";
+        }
+        if(missingParameter > 0) {
+            throw new IllegalArgumentException("Missing Parameter: " + missingParameterList);
+        }
+
+        String beatUrl = masterUrl + beatCommand;
+        String beatBody =  "{" +
+                "    \"id\": \"" + serverId + "\"," +
+                "    \"ipAddress\": \"" + serverIp + "\"," +
+                "    \"connected\": " + connected + "," +
+                "    \"port\": " + port +
+                "}";
```

Comment:
Same comment as in the client operator, I believe the body construction is an internal concern of the beat